### PR TITLE
Documentation for RutingAuto container provider

### PR DESCRIPTION
**Pull request by dantleech**
Documentation for the new routing auto container parameter provider.

### CODE REVIEW

File: bundles/routing_auto/token_providers.rst [hunk 1]
```diff
@@ -117,6 +117,39 @@ feature.
             </mapping>
         </auto-mapping>
 
+``container``
+-------------
+
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: you should add a versionadded directive telling that this was introduced in CMF 1.3

File: bundles/routing_auto/token_providers.rst [hunk 2]
```diff
@@ -117,6 +117,39 @@ feature.
...
+            <mapping class="Acme\ForumBundle\Document\Article" uri-schema="/{base_url}/good-day">
+                <token-provider token="base_url" name="container" >
+                    <option name="parameter">my_parameter.base_path</option>
+                </token>
+            </mapping>
+        </auto-mapping>
+
+.. note::
+
+    Parameters from the container will not be slugified. It is you
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: "you reponsibility" -> "your responsibility"

File: bundles/routing_auto/token_providers.rst [hunk 3]
```diff
@@ -117,6 +117,39 @@ feature.
...
+        # src/Acme/ForumBundle/Resources/config/cmf_routing_auto.yml
+        Acme\ForumBundle\Document\Article:
+            uri_schema: {base_url}/good-day
+            token_providers:
+                base_url: [container, { parameter: my_parameter.base_path }
+
+    .. code-block: xml
+
+        <?xml version="1.0" ?>
+        <!-- src/Acme/ForumBundle/Resources/config/cmf_routing_auto.xml -->
```
Comment (WouterJ) [inline bundles/routing_auto/token_providers.rst]: while it is invalid XML, you should put this comment before the doctype

**Comment by dantleech** [comment]
Updated. @WouterJ can you merge if ok?

**Comment by WouterJ** [comment]
Thanks @dantleech

**Pull request merged**